clusterloader2/pkg/measurement/common/service_creation_latency.go

```diff
@@ -44,17 +46,22 @@ import (
 
 const (
 	serviceCreationLatencyName           = "ServiceCreationLatency"
-	serviceCreationLatencyWorkers        = 10
 	defaultServiceCreationLatencyTimeout = 10 * time.Minute
 	defaultCheckInterval                 = 10 * time.Second
 	pingBackoff                          = 1 * time.Second
-	pingChecks                           = 10
 
 	creatingPhase     = "creating"
 	ipAssigningPhase  = "ipAssigning"
 	reachabilityPhase = "reachability"
 )
 
+var (
+	serviceCreationLatencyWorkers int
```

Comment:
Could you move this from global vars to serviceCreationLatencyMeasurement struct? This way mutliple meassurement invocation can specify their own values.